runtime/polkadot/src/weights/pallet_utility.rs

```diff
@@ -45,27 +45,24 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 	fn batch(c: u32, ) -> Weight {
-		(4_943_000 as Weight)
-			// Standard Error: 2_000
-			.saturating_add((3_798_000 as Weight).saturating_mul(c as Weight))
+		(11_289_000 as Weight)
```

Comment:
≈x2 jump

Reply:
The constant component does not really matter here compared to the component `c`.  
`c` goes up to 1000 for this benchmark. SWC will soon take the correct slope maxima into account to simplify these comparisions.